change/@microsoft-teams-js-b91fc37f-8189-4031-aec5-bedd6699cba1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
Just want to be explicit about this type "minor". Even though we have added required parameters to an existing interface, does the type still be minor? Is it because the scenario was never working before and is not eligible to be considered as a breaking change?

Reply:
Yeah, this is breaking the existing interface, but because this is a beta API we don't consider it a breaking change to the library